cmd/describe/machinepool/nodepool.go

```diff
@@ -0,0 +1,115 @@
+package machinepool
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	ocmOutput "github.com/openshift/rosa/pkg/ocm/output"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+func describeNodePool(r *rosa.Runtime, cluster *cmv1.Cluster, clusterKey string, nodePoolID string) error {
+	r.Reporter.Debugf("Fetching node pool '%s' for cluster '%s'", nodePoolID, clusterKey)
+	nodePool, exists, err := r.OCMClient.GetNodePool(cluster.ID(), nodePoolID)
+	if err != nil {
+		return err
+	}
+	if !exists {
+		r.Reporter.Infof("Machine pool '%s' not found", nodePoolID)
```

Comment:
Also here, we might want to error instead.

Reply:
changed